merlion/models/forecast/lstm.py

```diff
@@ -1,12 +1,24 @@
 #
-# Copyright (c) 2021 salesforce.com, inc.
+# Copyright (c) 2022 salesforce.com, inc.
 # All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 # For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 #
 """
 A forecaster based on a LSTM neural net.
 """
+try:
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+    from torch.utils.data import DataLoader, Dataset
+except ImportError as e:
+    err = (
+        "Try installing Merlion with optional dependencies using `pip install salesforce-merlion[torch]` or "
```

Comment:
Should this recommend `salesforce-merlino[deep-learning]` instead?

Reply:
Thanks for the catch. Ditto @ below.